fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1660,12 +1671,12 @@ private static void parseSearchResultParameter(Class<?> resourceType, FHIRSearch
             } else if (SearchConstants.SORT.equals(name) && first != null) {
                 // in R4, we only look for _sort
                 // Only first value is used, which matches behavior of other parameters that are supposed to be specified at most once
-                sort.parseSortParameter(resourceTypeName, context, first, lenient);
+                sort.parseSortParameter(resourceTypeName, context, first);
             } else if (name.startsWith(SearchConstants.INCLUDE) || name.startsWith(SearchConstants.REVINCLUDE)) {
-                parseInclusionParameter(resourceType, context, name, values, lenient);
+                parseInclusionParameter(resourceType, context, name, values, context.isLenient());
```

Comment:
any reason not to remove the lenient parameter from parseInclusionParameter as well?

Reply:
Fixed.